jobserver/urls.py

```diff
@@ -73,7 +73,7 @@
 
 
 api_urls = [
-    path("job-requests/", JobRequestAPIList.as_view()),
+    path("job-requests/", JobRequestAPIListCreate.as_view()),
```

Comment:
Why are you replacing the list view? What's going to happen to that?

Reply:
I'm not!  Have a look at the JobRequsetAPIListCreate view (it's tiny), hopefully the docstring explains, if not I'd consider that a bug 🙂

Reply:
Oh yes. Don't mind me, I see now.